test/exec.js

```diff
@@ -45,6 +45,15 @@ test('config.fatal and unknown command', t => {
   shell.config.fatal = oldFatal;
 });
 
+test('options.fatal = true and unknown command', t => {
+  const oldFatal = shell.config.fatal;
+  shell.config.fatal = false;
+  t.throws(() => {
+    shell.exec('asdfasdf', { fatal: true }); // could not find command
+  }, /asdfasdf/); // name of command should be in error message
```

Comment:
This isn't actually safe. If the assertion fails, we'll never reset `shell.config.fatal`. Can you file a TODO against me to fix this and the other spots? I might actually have a github issue filed for this somewhere...